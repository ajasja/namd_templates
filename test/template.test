colvar {
  name CAO_theta

  width 15.0
  lowerboundary -180
  upperboundary 180
  lowerWallConstant 0.2
  upperWallConstant 0.2

  angle {
    group1 { dummyAtom $dx $dy $dz }  
    group2 { atomnumbers $CAid  } ;#CA
    group3 { atomnumbers 204 }
    group4 { atomnumbers 207 }
  }
}

% if $abf {
abf {
  colvars chi1
  fullSamples	10000
}

% } else {
no abf

% }
