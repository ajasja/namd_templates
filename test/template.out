colvar {
  name CAO_theta

  width 15.0
  lowerboundary -180
  upperboundary 180
  lowerWallConstant 0.2
  upperWallConstant 0.2

  angle {
    group1 { dummyAtom 0 0 1 }  
    group2 { atomnumbers 202  } ;#CA
    group3 { atomnumbers 204 }
    group4 { atomnumbers 207 }
  }
}
abf {
  colvars chi1
  fullSamples	10000
}
